cmd/pipeline/destroy.go

```diff
@@ -124,7 +130,23 @@ func (pc *Command) ExecuteDestroyPipeline(ctx context.Context, opts *DestroyOpti
 	return nil
 }
 
-func (pc *Command) destroyPipeline(ctx context.Context, name, namespace string, destroyVolumes, destroyDependencies bool) (*types.GitDeployResponse, error) {
+func shouldDestroyDependencies(opts *DestroyOptions) bool {
+	isForceRedeployDependenciesSetInOktetoInstance := env.LoadBoolean(constants.OktetoForceDestroyDependencies)
```

Comment:
We don't have yet the code in the CLI to set this `OktetoForceDestroyDependencies` based on the server information, right?

Reply:
This variable comes from the backend in the same way as `OKTETO_FORCE_REMOTE` does for the remote flag